src/main.rs

```diff
@@ -15,10 +15,12 @@ mod flat_writer;
 mod json_writer;
 
 use zip_info::WriteZipInfo;
+use zip_info::Args;
+use zip_info::StatArgs;
```

Comment:
We can move all of these on one line using the shortcut described [here](https://doc.rust-lang.org/book/crates-and-modules.html#importing-modules-with-use).